requirements.txt

```diff
@@ -1,5 +1,6 @@
 geopy>=1.11.0
+numpy==1.13.1
+scipy==0.19.0
 pycpfcnpj==1.0.2
 scikit-learn==0.18.1
-scipy==0.19.0
```

Comment:
Can we keep `requirements.txt` sorted?

Reply:
During the scikit-learn wheel build it was failing because of "missing scipy". After changing the order it was building with no error. I can retry and see if was something in that moment.